sliders/views.py

```diff
@@ -16,19 +16,13 @@ def view(request):
     connection = psycopg2.connect(
         host=db_params.hostname, database=db_params.path[1:],
         user=db_params.username, password=db_params.password
-    )# psycopg2.connect(db_connection)
+    )
         # creating the cursor (vessel to the db)
     cursor=connection.cursor()
         # executing the query
-    # cursor.execute("SELECT * FROM public.agric_indicator where gs_id<10 ")
-#     cursor.execute('''select ppp_sum,(ndvi_jfm+ndvi_amj+ndvi_jas+ndvi_ond)/4,
-# (ndwi_jfm+ndwi_amj+ndwi_jas+ndwi_ond)/4,soil_phos,soil_copper,soil_iron,soil_magne,
-# rainfall,soil_alumi,soil_potas,(lst_jfm+lst_amj+lst_jas+lst_ond)/4,slope,elevation,
-# land_cover, gs_id from public.agric_indicator where gs_id<10 ''')
-
-    cursor.execute('''select ppp_sum,soil_copper,soil_phos,soil_potas,soil_boron,soil_alumi,soil_iron,soil_magne,ndwi_jfm,rainfall,
-    ndvi_jfm,land_cover,slope,elevation,lst_jfm,
-     gs_id from public.agric_indicator where gs_id<24320''')
+
+    cursor.execute('''select ppp_sum,soil_copper,soil_phos,soil_potas,soil_boron,soil_alumi,soil_iron,soil_magne,ndwi_jfm,presp_jfm,
```

Comment:
@lambert3 , seeing as this is the only significant change in this PR, I've raised an equivalent one here -> https://github.com/Geo-Gecko/ASRT-B/pull/4/files. Please see if there is anything left out and make a comment on this thread. If not, a thumbs up will suffice.

Reply:
@RonKbS , i have checked out the pull request above and it's very okay. Otherwise, I will make my PRs more specific